docs/src/troubleshooting.md

```diff
@@ -15,13 +15,11 @@ If the server is crashing on startup, try running `LSP: Troubleshoot server` fro
 
 ## Updating the PATH used by LSP servers
 
-You can confirm that your issue is due to `PATH` being different by starting Sublime Text from the command line so that it inherits your shell's environment.
+Sublime Text might see a different `PATH` from what your shell environment uses and might not be able to find the server binary due to that. You can see what ST thinks your `PATH` is by opening the ST console by clicking on *View > Show Console*, and running `import os; os.environ["PATH"]` in that console.
```

Comment:
The old text started with "You can confirm that your issue is due..." even though there is no any issue in this context.